sources/core/Stride.Core.CompilerServices/Analyzers/STRDIAG007DataMemberOnDelegate.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Stride.Core.CompilerServices.Common;
+
+namespace Stride.Core.CompilerServices.Analyzers;
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+internal class STRDIAG007DataMemberOnDelegate : DiagnosticAnalyzer
+{
+    public const string DiagnosticId = "STRDIAG007";
+    private const string Title = "Invalid DataMemberAttribute";
+    private const string MessageFormat = "Invalid DataMembermode for the specified member '{0}'. A Delegate can't be a Delegate.";
```

Comment:
`A Delegate can't be a Delegate.` -> `A Delegate is not serializable`.

Reply:
[5340232](https://github.com/stride3d/stride/pull/1864/commits/534023297d2e48c5c96314de51adba6da0023b1c)
changed the error message.